src/.nuget/optdata/ibcmerge.csproj

```diff
@@ -9,8 +9,24 @@
     <PackageReference Include="microsoft.dotnet.ibcmerge" Version="[$(IbcMergePackageVersion)]" Condition="'$(IbcMergePackageVersion)'!=''" />
   </ItemGroup>
 
+  <!--                                                                       -->
+  <!-- Task: DumpIbcMergePackageVersion                                       -->
+  <!--                                                                       -->
+  <!-- Notes:                                                                -->
+  <!--                                                                       -->
+  <!-- DumpIbcMergePackageVersion is used by build.sh and build.cmd to pass  -->
+  <!-- the version information to cmake. The task will write a file to the   -->
+  <!-- intermediates directory to be read back by build.cmd/sh. The path for -->
+  <!-- the file is IbcMergePackageVersionOutputFile                          -->
+  <!--                                                                       -->
   <Target Name="DumpIbcMergePackageVersion">
-    <Message Importance="high" Text="$(IbcMergePackageVersion)" />
+    <!-- Error if PgoDataPackageVersionOutputFile is not set. -->
+    <Error Condition="'$(IbcMergePackageVersionOutputFile)'==''" />
+
+    <!-- Cleanup old version file -->
+    <Delete Files="$(IbcMergePackageVersionOutputFile)" Condition="Exists('$(IbcMergePackageVersionOutputFile)')" />
+    <Message Text="IbcMergePackageVersion: $(IbcMergePackageVersion) written to: $(IbcMergePackageVersionOutputFile)" Importance="High" />
```

Comment:
nit: either `written to` -> `will be written to` or move `Message` to after `WriteLinesToFile`.